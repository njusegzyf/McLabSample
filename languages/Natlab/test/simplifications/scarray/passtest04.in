
if (A | B) & (C | D)
    foo();
end
