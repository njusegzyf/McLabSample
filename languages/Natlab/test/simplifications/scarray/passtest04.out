
t4 = A;
if t4
    t2 = t4;
else
    t5 = B;
    t2 = t4 | t5;
end
if t2
    t6 = C;
    if t6
        t3 = t6;
    else
        t7 = D;
        t3 = t6 | t7;
    end
    t1 = t2 & t3;
else
    t1 = t2;
end
if t1
    foo();
end
